apps/frontend/src/components/drinkAction-drink-list.tsx

```diff
@@ -0,0 +1,29 @@
+import { Card, CardContent } from '@/components/ui/card';
+import { Carousel, CarouselContent, CarouselItem } from '@/components/ui/carousel';
+import { EventDetails } from '@/models/event';
+
+interface Props {
+  event: EventDetails;
+}
+
+export function DrinkList({ event }: Props) {
+  return (
+    <Carousel className='w-[51rem]'>
+      <CarouselContent className='-ml-1'>
+        {event.drinkActions.map((drinkAction) => (
```

Comment:
I think we shouldn't show every drink action here (though it might work with the carousel), would be better to show some stats, like most consumed drinks